drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/test/java/org/drools/workbench/screens/dtablexls/backend/server/conversion/DTCellValueUtilitiesTest.java

```diff
@@ -36,6 +36,7 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 
+@org.junit.Ignore
```

Comment:
Why is this now ```@Ignore```'ed?

Reply:
Oops :-( I accidentally committed this. It's fixed now.